CONTRIBUTING.adoc

```diff
@@ -50,101 +45,98 @@ https://github.com/openshift/origin[origin repo].
 === Develop locally on your host
 
 You can develop OpenShift 3 on Windows, Mac, or Linux, but you'll need Docker
-installed on Linux to actually launch containers.
+installed on Linux to actually launch containers. Client and server binaries
+can be built locally or in the `openshift/release` container environemnt. The
+http://golang.org/[Go] programming language is only necessary for building on
+the local host.
```

Comment:
@smarterclayton for dev build env definitively good to have and use it, for production build definitively not needed, as you hinted it is not always needed and it is not needed to do build, but it should be used for development/by developers(as many other things...), In my opinion question is more what would be best(better) way to provide easy setup for Go development environment on Fedora(or other distributions)
@stevekuznetsov if it does come from RHEL rebuild, it should be the same as in Fedora, no idea if not, but if GOROOT lives in same location it should be write protected from regular users too